api/src/test/java/org/apache/iceberg/expressions/TestNumericLiteralConversions.java

```diff
@@ -159,12 +148,9 @@ public void testDoubleToDecimalConversion() {
     assertThat(lit.to(Types.DecimalType.of(9, 1)).value())
         .as("Value should round using HALF_UP")
         .isEqualTo(new BigDecimal("34.6"));
-    assertThat(lit.to(Types.DecimalType.of(9, 2)).value())
-        .as("Value should match")
-        .isEqualTo(new BigDecimal("34.56"));
-    assertThat(lit.to(Types.DecimalType.of(9, 4)).value())
-        .as("Value should match")
-        .isEqualTo(new BigDecimal("34.5600"));
+    assertThat(lit.to(Types.DecimalType.of(9, 2)).value()).isEqualTo(new BigDecimal("34.56"));
+
```

Comment:
nit: can we remove this unnecessary newline?

Reply:
removed the new line